src/app/screens/coinDashboard/coinHeader.tsx

```diff
@@ -355,6 +355,7 @@ export default function CoinHeader(props: CoinBalanceProps) {
         <RowContainer>
           <BalanceTitleText>{getDashboardTitle()}</BalanceTitleText>
           {coin === 'brc20' && <ProtocolText>BRC-20</ProtocolText>}
+          {fungibleToken?.protocol === 'stacks' && <ProtocolText>SIP-10</ProtocolText>}
```

Comment:
Faithful to the original implementation: https://github.com/secretkeylabs/xverse-web-extension/pull/735/files#diff-0f2d59a78ff9c8f36e699fcce53c5418f5aeaa153b10dfa3febe156dea9c2e0aL369

If we go by `coin`, it would yield `FT`. I don't think there's an edge case where both conditions are true simultaneously, so I avoided nested ternary operators.